File: C++17/백준/Bronze/2525. 오븐 시계/오븐 시계.cc
#include<iostream>
using namespace std;

int main() {
    int hour, min, turn, time;
    cin >> hour >> min;
    cin >> turn;
   
    time = hour * 60 + min + turn;

    hour = time / 60;
    min = time % 60;

    while (1) {
        if (hour < 24)
            break;
        hour -= 24;
    }

    cout << hour << " " << min << endl;

}

File: Python/백준/Bronze/2525. 오븐 시계/오븐 시계.py
a,b = map(int,input().split())
c = int(input())

if a==0:
    a=24

min = a*60 + b + c
hour = min // 60
min_result = min % 60

if hour >= 24:
    hour -= 24

print(hour,min_result)
